docs/_articles/en/einstein/einstein-troubleshoot.md

```diff
@@ -5,4 +5,33 @@ lang: en
 
 Generative AI uses large language models (LLMs) to generate the output. Because LLMs are trained by other organizations, the output isn’t always what you expect, and the code that’s generated isn’t perfect. If the output doesn’t meet your expectations, you can generate a new output by starting over. The generated output from the previous attempt isn’t saved, and the new output replaces it.
 
-To keep track of all known issues and potential workarounds, use the [Issues](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) tab in this repo.
+To track all known issues and potential workarounds, use the [Issues](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) tab in this repo.
+
+**Note:** If your company requires the use of a proxy, we recommend setting this through the Salesforce CLI, not through VS Code, in order to use the Einstein for Developers extension.
+
+## Troubleshooting Checklist
+Use this troubleshooting checklist to identify and eliminate common problems with using Einstein for Developers.
+
+### Installation Checklist
+
+1. Confirm that you have the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) or the [Salesforce Expanded Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-expanded) installed.
+2. Check that you are on VS Code version 1.82.0 or higher. 
+3. Check that your Apex Extension is version 59.8.0 or higher.
```

Comment:
If the User is below the min required version of Apex I think they see a pop-up notification telling them as much.  Have we seen cases where users are on an older version and haven't seen the pop up notifications?  Maybe we add a comment here that says something like "you should be seeing a notification on startup if this is the case"

Reply:
I think if they have an old version installed they see the error, but if they're missing it entirely that popup might not show. I do recall being on a customer call where it was missing, but it was on windows and they were missing a lot of the prereqs. They also won't see the notification for this if it bombs out before then, like missing the core extensions or the wrong version of it.